src/reaper_osara.cpp

```diff
@@ -3018,8 +3022,8 @@ void cmdIoMaster(Command* command) {
 void cmdReportRippleMode(Command* command) {
 	postCycleRippleMode(command->gaccel.accel.cmd);
 }
-
-string formatTracksWithState(const char* prefix, TrackStateCheck checkState,
+template <typename Func>
```

Comment:
Neat! :)

Reply:
I didn't know whether to go with a template or a function object parameter. Probably doesn't make much difference, but do you have a preference?

Reply:
I don't have a preference here. This is fine.